specification/edgeorder/resource-manager/Microsoft.EdgeOrder/preview/2020-12-01-preview/edgeorder.json

```diff
@@ -297,13 +297,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrder/orderCollections": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrder/orders": {
```

Comment:
Changing the name of a tracked resource type is equivalent to deprecating the existing resource type.  Please ensure that all existing instances of this resource type have been deleted from the ARM Resource Cache.  https://armwiki.azurewebsites.net/introduction/concepts/caching.html

Reply:
@pilor As I mentioned early, We are preview service and we only have less than 20 resources in our internal BVT environment. We ll follow the link mentioned about and ensure ARM rsource cache is cleaned